pom.xml

```diff
@@ -66,7 +107,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <deploy.projectId>nina-laura-dagm-step-2020</deploy.projectId>
+          <deploy.projectId>dagmawi-step-2020</deploy.projectId>
```

Comment:
Does our project ID not work for your testing? Just curious

Reply:
I think when I pulled it reverted it

Reply:
Please change this.